src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  color: #3e4e61;
+  font-weight: 300;
+}
+
+.formForBig {
+  position: relative;
+}
+
+.formForSmall {
+  position: relative;
+}
+
+.bigSearch {
+  position: absolute;
+  top: 37px;
+  left: 26px;
+}
+
+.smallSearch {
+  height: 11px;
+  width: 11px;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+}
+
+.search {
```

Comment:
Add correct font-weight for all inputs and correct font-weight for placeholders (use ::placeholder)